hxinsight-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-hxinsight-extension/module.properties

```diff
@@ -15,12 +15,12 @@
 
 # Note. These properties are filtered at build time by Maven, single sourcing from POM properties
 module.id=${project.artifactId}
-#module.aliases=myModule-123, my-module
+module.aliases=hxinsight-extension, repository-hxinsight-extension
 module.title=${project.name}
 module.description=${project.description}
 module.version=${project.version}
 
 # The following optional properties can be used to prevent the module from being added
 # to inappropriate versions of the WAR file.
-module.repo.version.min=7.1
-#module.repo.version.max=7.1
+module.repo.version.min=7.3.3
```

Comment:
Why 7.3.3 rather than 7.3.0?

Reply:
Same question just came to my mind - fixed

Reply:
Firstly the indication for me was the `acs-packaging` where I'm adding the JAMP to branch `7.3.N`, which effectively will be `7.3.3` after release. Later, after few minutes came to my mind, that `hxi-connector` should support `7.3.x` in general